WriteThrough-/ListTriggerSync.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Cosmos;
+using System.Collections;
+using System.ComponentModel;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Redis;
+using System.Collections.Generic;
+using StackExchange.Redis;
+using System.Linq;
+using Microsoft.Azure.Cosmos.Linq;
+using Container = Microsoft.Azure.Cosmos.Container;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Redis.Samples
+{
+    public record ListData
+    (
+        string id,
+        List<string> value
+    );
+    public static class WriteThrough
+    {
+        //redis connection string
+        public const string localhostSetting = "redisLocalhost";
+        private static readonly IDatabase cache = ConnectionMultiplexer.Connect(Environment.GetEnvironmentVariable(localhostSetting)).GetDatabase();
+
+        //connecting to CosmosDB
+        public const string Endpoint = "Endpoint";
+
+        [FunctionName(nameof(ListTrigger))]
+        public static void ListTrigger(
+            [RedisListTrigger(localhostSetting, "listTest")] string listEntry, [CosmosDB(
+                            databaseName: "databaseName",
```

Comment:
I think these can be defined on line 27 after the connection strings are define, is that correct?